apps/web/src/views/Tokens/SendTokensForm.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  Stack,
+} from "@chakra-ui/react";
+import {
+  type FA12TokenBalance,
+  type FA2TokenBalance,
+  tokenDecimals,
+  tokenSymbolSafe,
+} from "@umami/core";
+import repeat from "lodash/repeat";
```

Comment:
IIRC we prohibit default imports somewhere 🤔

Reply:
I'm not sure why do we prohibited default imports from 3rd party libs?